packages/flutter/test/cupertino/button_test.dart

```diff
@@ -612,32 +680,57 @@ void main() {
     expect(focusNode.hasFocus, isFalse);
   });
 
-  testWidgets('IconThemeData is not replaced by CupertinoButton', (WidgetTester tester) async {
-    const IconThemeData givenIconTheme = IconThemeData(size: 12.0);
+  testWidgets('IconThemeData falls back to default value when the TextStyle has a null size', (WidgetTester tester) async {
+    const IconThemeData defaultIconTheme = IconThemeData(size: kCupertinoButtonDefaultIconSize);
 
     IconThemeData? actualIconTheme;
 
+    // large size
     await tester.pumpWidget(
       CupertinoApp(
+        theme: const CupertinoThemeData(
+          textTheme: CupertinoTextThemeData(
+            actionTextStyle: TextStyle(),
+          ),
+        ),
         home: Center(
-          child: IconTheme(
-            data: givenIconTheme,
-            child: CupertinoButton(
-              onPressed: () {},
-              child: Builder(
-                  builder: (BuildContext context) {
-                    actualIconTheme = IconTheme.of(context);
+          child: CupertinoButton(
+            onPressed: () {},
+            child: Builder(
+                builder: (BuildContext context) {
+                  actualIconTheme = IconTheme.of(context);
 
-                    return const Placeholder();
-                  }
-              ),
+                  return const Placeholder();
+                }
             ),
           ),
         ),
       ),
     );
+    expect(actualIconTheme?.size, defaultIconTheme.size);
+
+    // small size
```

Comment:
Nit: Comments should start with a capital letter and end with a period, as per the style guide.